third_party/bootstrap-material-design/.gitignore

```diff
@@ -0,0 +1,22 @@
+# https://git-scm.com/docs/gitignore
```

Comment:
Ok, thanks.

Let's not create a pipeline, but let's only submit the license and the _required_ files to build the forseti website.